DESCRIPTION

```diff
@@ -18,28 +19,25 @@ Depends:
     R (>= 4.1.0)
 Imports:
     cli,
+    purrr,
+    R.utils,
+    R6,
     reticulate,
+    rlang,
+    rstudioapi,
```

Comment:
I think rstudioapi is only only used in combination with `requireNamespace`, so this should not be an Imports, no?